src/java.base/share/classes/java/security/PEM.java

```diff
@@ -29,30 +29,48 @@
 
 import sun.security.util.Pem;
 
+import java.io.InputStream;
+import java.util.Base64;
 import java.util.Objects;
 
 /**
- * {@code PEMRecord} is a {@link DEREncodable} that represents Privacy-Enhanced
- * Mail (PEM) data by its type and Base64 form.  {@link PEMDecoder} and
- * {@link PEMEncoder} use {@code PEMRecord} when representing the data as a
- * cryptographic object is not desired or the type has no
- * {@code DEREncodable}.
+ * {@code PEM} is a {@link DEREncodable} that represents Privacy-Enhanced
+ * Mail (PEM) data by its type and Base64 content.
  *
- * <p> {@code type} and {@code content} may not be {@code null}.
- * {@code leadingData} may be null if no non-PEM data preceded PEM header
- * during decoding.  {@code leadingData} may be useful for reading metadata
- * that accompanies PEM data.
+ * <p> The {@link PEMDecoder#decode(String)} and
+ * {@link PEMDecoder#decode(InputStream)} methods return a {@code PEM} object
+ * when the data type cannot be represented by a cryptographic object.
+ * If {@code PEM} is desired instead of a cryptographic object, the
+ * decoding methods {@link PEMDecoder#decode(String, Class)} or
+ * {@link PEMDecoder#decode(InputStream, Class)} are used with
+ * {@code PEM.class} as an argument.
  *
- * <p> No validation is performed during instantiation to ensure that
- * {@code type} conforms to {@code RFC 7468}, that {@code content} is valid
- * Base64, or that {@code content} matches the {@code type}.
- * {@code leadingData} is not defensively copied and does not return a
- * clone when {@linkplain #leadingData()} is called.
+ * <p> A {@code PEM} object can be encoded back to its textual format by using
+ * encode methods in {@link PEMEncoder} or the {@link #toString()} method.
```

Comment:
I'd probably say "calling" instead of "using" and switch the order, and a few other tweaks. Suggest: 

"A {@code PEM} object can be encoded back to its textual format by calling the {@link #toString()} method or the encode methods of {@link PEMEncoder}."